src/Commands/Administration/language.js

```diff
@@ -0,0 +1,69 @@
+// ██████ Integrations █████████████████████████████████████████████████████████
+
+// —— Import base command
+const Command = require("../../Base/Command");
+
+// ██████ | ███████████████████████████████████████████████████████████ | ██████
+
+// —— Create & export a class for the command that extends the base command
+class Language extends Command {
+
+    constructor(client) {
+        super(client, {
+            name        : "language",
+            description : "Change the language used by Luna in the guild",
+            usage       : "£language",
+            exemple     : [],
+            args        : false,
+            category    : "Administration",
+            cooldown    : 10000,
+            aliases     : ["sl"],
+            permLevel   : 9,
+            permission  : "ADMINISTRATOR",
+            allowDMs    : false
+        });
+
+    }
+
+    async run(message, args) {
+
+        const client = this.client;
+
+        // —— Retrieve the language information for this command
+        const lang = client.language.get(message.guild.local).language();
+
+        const selector = await super.respond({embed : {
+            description : lang[0]
+        }});
+
+        ["🇬🇧", "🇫🇷"].forEach((e) => selector.react(e));
+
+        const filter = (reaction, user) => {
+            return ["🇬🇧", "🇫🇷"].includes(reaction.emoji.name) && user.id === message.author.id;
+        };
```

Comment:
I know it's a comment from a random person but I think it would be smart to create a variable with the 2 emotes in it. This would help you and future contributors if you wanted to add more languages because you would only need to edit the variable not all instances of the array (and it looks cleaner imo). :)

Reply:
Indeed, I'm stupid not to have thought about it, thank you very much for your help, it's not a random comment, it's a comment that helps me, so thank you!